llvm/tools/llvm-exegesis/README.md

```diff
@@ -13,8 +13,16 @@ Analysis mode in `llvm-exegesis` is supported on all platforms on which LLVM is.
 #### Currently Supported Operating Systems for Benchmarking
 
 Currently, `llvm-exegesis`  only supports benchmarking on Linux. This is mainly
-due to a dependency on the Linux perf subsystem for reading performance
-counters.
+due to a dependency on the Linux perf subsystem and libpfm4 for reading
```

Comment:
This should probably go in the LLVM CMake documentation. This page isn't intended to provide detailed build directions, and people are more likely to look there.